users.js

```diff
@@ -773,23 +773,6 @@ class User {
 			return;
 		}
 
-		if (Config.tokenhosts) {
-			let host = tokenDataSplit[4];
-			if (Config.tokenhosts.length === 0) {
-				Config.tokenhosts.push(host);
-				console.log('Added ' + host + ' to valid tokenhosts');
-				require('dns').lookup(host, (err, address) => {
-					if (err || (address === host)) return;
-					Config.tokenhosts.push(address);
-					console.log('Added ' + address + ' to valid tokenhosts');
-				});
-			} else if (Config.tokenhosts.indexOf(host) < 0) {
```

Comment:
I suppose so - guess I looked through too quickly since `Config.tokenhost` isn't even in the example config anymore and that we have a challenge-response system.

The reason then changes to quote e3fdd4b4fc2d29e480ef23dd909dc08b16f3f35b:
>With the advent of challenge-response authentication, the `tokenhosts` feature is no longer necessary.
which is the second reason why I removed it